app/Filament/Resources/DatabaseHostResource/RelationManagers/DatabasesRelationManager.php

```diff
@@ -48,18 +56,29 @@ public function table(Table $table): Table
         return $table
             ->recordTitleAttribute('servers')
             ->columns([
-                TextColumn::make('database')->icon('tabler-database'),
-                TextColumn::make('username')->icon('tabler-user'),
-                TextColumn::make('remote'),
+                TextColumn::make('database')
+                    ->hidden(fn () => !auth()->user()->can('viewList database'))
```

Comment:
Isn't there a nicer way to hide the whole table instead of hiding every single column?

Reply:
So it's a relation manager. Not a tab, so it's rendered as it's own table. Just like allocations on the edit node/server page